app/editor/src/features/content/form/ContentForm.tsx

```diff
@@ -24,7 +27,8 @@ export const ContentForm: React.FC = () => {
   const navigate = useNavigate();
   const { id } = useParams();
   const [{ mediaTypes }] = useLookup();
-  const [, { getContent, addContent, updateContent }] = useContent();
+  const [, { getContent, addContent, updateContent, deleteContent }] = useContent();
+  const { isShowing, toggle } = useModal();
 
   const [active, setActive] = React.useState('properties');
   const [mediaTypeOptions, setMediaTypeOptions] = React.useState<IOptionItem[]>([]);
```

Comment:
I've stopped adding options to state lately.  Not certain what the best approach is, but if they're not going to change it seems better to not add them to state.